resharper/src/resharper-unity/Rider/UnityOptionsPage.cs

```diff
@@ -28,6 +28,10 @@ public class UnityOptionsPage : OptionsPageBase
 
             CheckBox((UnitySettings s) => s.InstallUnity3DRiderPlugin, "Install or update Rider plugin automatically");
             CheckBox((UnitySettings s) => s.AllowAutomaticRefreshInUnity, "Automatically refresh Assets in Unity");
+            
+            Header("UI");
```

Comment:
This is going the way of having too many checkboxes. And we're still hiding NuGet window for example. Let's just make one option which configures whether we are in a simplified view or not. @kskrygan

Reply:
If you look at requests, you will see that one person requested "Solution Configuration" and multiple others - "Database". And showing Nuget would be misleading/harmful, because there is absolutely no chance it may work in the generated solution.